src/AccessibilityInsights.Actions/Actions/ListenAction.cs

```diff
@@ -23,10 +19,8 @@ namespace Axe.Windows.Actions
     [InteractionLevel(UxInteractionLevel.NoUxInteraction)]
     public class ListenAction:IDisposable
     {
-        RecorderSetting Config { get; set; }
         ElementContext ElementContext { get; set; }
         EventListenerFactory EventListener { get; set; }
-        List<IA11yEventMessage> EventRecords { get; set; }
```

Comment:
EventRecords is no longer maintained by ListenAction - we provided no direct getters previously, so I've changed it so the UI relies directly on its list.

Reply:
Can we make the setters internal or private?

---
In reply to: [275580493](https://github.com/Microsoft/accessibility-insights-windows/pull/303#discussion_r275580493) [](ancestors = 275580493)